src/components/config_profile/include/config_profile/profile.h

```diff
@@ -552,6 +552,11 @@ class Profile : public protocol_handler::ProtocolHandlerSettings,
    */
   size_t update_before_hours() const;
 
+  /**
+   * @brief Return security level that will be configured in the OpenSSL
+   */
+  uint32_t security_level() const;
```

Comment:
@VadymLuchko OVERRIDE?

Reply:
In this case security_level() - public, non virtual method of Profile class.
We also have method with the same name in class CryptoManagerSettings:
virtual uint32_t security_level() const = 0;

And overridden method in CryptoManagerSettingsImpl (derived from  CryptoManagerSettings):
  uint32_t security_level() const OVERRIDE {
    return profile_.security_level();
  }
  
  
![uml](https://user-images.githubusercontent.com/76956836/113132842-a84e0f80-9227-11eb-8f96-e77600f3c2ee.png)